MyReadingList/MyReadingList/MyReadingListView.swift

```diff
@@ -10,21 +10,19 @@ import SwiftUI
 
 struct MyReadingListView: View {
     var body: some View {
-        VStack {
-            Text(NSLocalizedString("ui.myReadingList.title", value: "My Reading List", comment: "title for the MyReadingList screen"))
-                .font(.headline)
-            Spacer()
-            
-            List {
-                HStack{
-                    Button {
-                        //do nothing for now
+        NavigationView {
+            VStack {
+                List {
+                    HStack{
+                        NavigationLink {
```

Comment:
It looks like the code for this NavigationLink is not all properly indented

Reply:
Thanks a lot for pointing this. I find a shortcut of indenting my code: select the code that need to be indented (have been charged) and press Control + I

Reply:
Excellent find!